packages/core/core/src/utils/startup-logger.ts

```diff
@@ -1,9 +1,36 @@
 import chalk from 'chalk';
 import CLITable from 'cli-table3';
 import _ from 'lodash/fp';
+import path from 'node:path';
 
 import type { Core } from '@strapi/types';
 
+const formatDbInfo = (app: Core.Strapi) => {
```

Comment:
Could you expose a db.getDatabaseInfo() or sth like that to abstract the DB implementation details and just return the basinc info (also thinking about the PR you just made, the less we use the connection object directly outside the better)
Even if those are already accessed somewherese else I think it would be nice to start having better encapsulation :)